### PR TITLE
Code review suggestions

### PR DESCRIPTION
**Pull request by beaslecd**
Description of project:
Allows users to take pictures or randomly grab a photo from the web and creates a puzzle game out of the pictures. 

What I've learned from this code review:
- I learned about visability modifiers and how to properly use them.
- I learned how to take a random photo from the web and convert it into a sliding pic
- I learned about correct formatting of Kotlin 

Was the program available in UC Github on time? Yes.
Is the program documented/commented well enough for you to understand? Yes.
Does the program compile? Yes.

### CODE REVIEW

File: app/src/main/java/com/example/puzzlepic/dto/Picture.kt [hunk 1]
```diff
@@ -5,8 +5,14 @@ import com.google.gson.annotations.SerializedName
...
-data class Picture(@SerializedName("id") var pictureID : String = "", @SerializedName("alt_description") var pictureTitle : String, @SerializedName("created_at") var date : String, var location : String, var notes : String,
-                        var puzzleID : Int = 0, @SerializedName("urls") var urls : UnsplashPhotoUrls ){
+data class Picture(
+    @SerializedName("id") var pictureID : String = "",
+    @SerializedName("alt_description") var pictureTitle : String,
+    @SerializedName("created_at") var date : String,
+    var location : String,
+    var notes : String,
+    var puzzleID : Int = 0,
+    @SerializedName("urls") var urls : UnsplashPhotoUrls ){
```
Comment (beaslecd) [inline app/src/main/java/com/example/puzzlepic/dto/Picture.kt]: Longer headers should be formated in individual lines. Reference: https://kotlinlang.org/docs/coding-conventions.html#class-headers

File: app/src/main/java/com/example/puzzlepic/service/PictureService.kt [hunk 2]
```diff
@@ -23,7 +23,7 @@ class PictureService {
      * Uses a callback to retrieve json data from network and handles response accordingly
      * @return an LiveArrayList of pictures.
      */
-    fun fetchPicture() : MutableLiveData<ArrayList<Picture>> {
+   internal fun fetchPicture() : MutableLiveData<ArrayList<Picture>> {
```
Comment (beaslecd) [inline app/src/main/java/com/example/puzzlepic/service/PictureService.kt:26]: Adding a visability modifier. Reference: https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces

File: app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt [hunk 3]
```diff
@@ -29,15 +29,8 @@ class MainFragment : Fragment() {
     private val IMAGE_GALLERY_REQUEST_CODE: Int = 2001
     private val SAVE_IMAGE_REQUEST_CODE: Int = 1999
     private val CAMERA_REQUEST_CODE: Int = 1998
-
-    val CAMERA_PERMISSION_REQUEST_CODE = 1997
-
+    private val CAMERA_PERMISSION_REQUEST_CODE = 1997
```
Comment (beaslecd) [inline app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt:44]: Adding a visability modifier and doing soem housekeeping in the variable section to make it neat. Reference: https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces

**Comment by discospiff** [comment]
Some good suggestions here.  These can either be merged or implemented manually.

**Comment by erswagner7** [comment]
Accepting code review changes from Sprint 1

**Pull request merged**